supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,70 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
+* Do not use JavaScript links. 
+* Do not use Flash links.
+* Do not use links in Frames or iframes.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+
+
+.Link text best practices
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location. 
+* Avoid irrelevant link text.
+* You can use a sentence or sentence fragment as the link text.
+
+
+
 [[internal-links]]
 == Internal links
 
+Internal links are links within www.redhat.com.
+
+.Example AsciiDoc: Internal links
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+To link to another guide or to a user story within the same guide:
+----
+For more information about <topic>, see link:<link>[<link_text>] in _<title_name>_.
+----
+
+[[rh-kb-links]]
+=== Linking to Red Hat Knowledgebase articles
+
+* Use the title of the Knowledgebase article for the link text, or use descriptive running text.
+* Call out that this is a Knowledgebase article when _not_ using running text.
+* When the link appears in *Additional resources*, put the article title first, followed by `(Red Hat Knowledgebase)`.
+
+
+.Example: Article title
+
+For a non-cloud environment, you can resize the disk and file system. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/199573[Does RHEL 7 support online resize of disk partitions?].
+
+.Example: Running text
+
+If your Apache web server configuration enables SSL security, verify that you enable only the TLSv1 protocol and disable SSLv2 and SSLv3. This is because of the link:https://access.redhat.com/solutions/1232413[POODLE SSL vulnerability (CVE-2014-3566)].
```

Comment:
> verify that you enabled only the TLSv1 protocol and disabled SSLv2 and SSLv3.

Does this read better? One can only verify a completed action?

Reply:
Don't know what moment in time this is supposed to represent, but either:

(verifying a past action) ...verify that you enabled only the ... and disabled the...

(in the middle of making this config decision) enable only the ... and disable the...